src/_data/sites/ja.yml

```
@@ -0,0 +1,542 @@
+    contributors. All rights reserved. The <a href="https://openjsf.org">OpenJS 
+    Foundation</a> has registered trademarks and uses trademarks.  For a list of 
+    trademarks of the <a href="https://openjsf.org">OpenJS Foundation</a>, please 
+    see our <a href="https://trademark-policy.openjsf.org">Trademark Policy</a> 
+    and <a href="https://trademark-list.openjsf.org">Trademark List</a>.  
+    Trademarks and logos not indicated on the 
+    <a href="https://trademark-list.openjsf.org">list of OpenJS Foundation 
+    trademarks</a> are trademarks&trade; or registered&reg; trademarks of 
+    their respective holders. Use of them does not imply any affiliation with 
+    or endorsement by them.
+  links:
+    open_jsf: The OpenJS Foundation
+    terms: 利用規約
+    privacy: プライバシーポリシー
+    bylaws: OpenJS Foundation 定款
+    trademark: 商標ポリシー
+    trademark_list: 商標リスト
+    cookies: クッキーポリシー
+
+#------------------------------------------------------------------------------
+# 404 Page
+#------------------------------------------------------------------------------
+
+404_page:
+  title: 404エラー
+  subtitle: ページが見つかりません
+  description: 申し訳ございませんが、お探しのページは移動または削除された可能性があります。
+  actions:
+    back_to_home: ホームページへ
+    browse_docs: ドキュメントを読む
+
+#------------------------------------------------------------------------------
+# Branding Page
+#------------------------------------------------------------------------------
+
+branding_page:
+  title: ブランディング
+  description: 正しいロゴの使い方
+  sections:
+    name:
+      title: 名前
+      description: >
+        ESLintと書くときは、EとSとLは必ず大文字を使用してください。「ES」はECMAScriptの略称、Lは「Lint」の頭文字です。
+    logo:
+      title: ロゴ
+      description:
+        - >
+          ESLintのロゴは、様々な背景に配置することができます。十分な縦・横の余白が必要です。
+        - >
+          隙間からなる内側の六角形の大きさの2倍が、ロゴとロゴタイプの間隔、およびロゴの周囲の余白の
+          大きさの目安になります。
+        - >
+          我々のロゴは汎用性があります。様々なブランドカラーに使用することができ、白以外の背景では透過させることで
+          明るい色を模倣することができます。ESLintのロゴは、ロゴタイプを付けずに単独で使用することもできますが、
+          可能であればESLintのロゴとロゴタイプの両方を使用することが望ましいです。
+      download_svg: SVG ダウンロード
+      download_png: PNG ダウンロード
+      change_background: "背景色の変更"
+    color_palette:
+      title: カラーパレット
+      description:
+        - >
+          ロゴや文言と同じくらい、ESLintのカラーパレットにはブランドを表す力があります。
+          デザインの見た目に影響を与えるだけでなく、感情を呼び起こし、
+          ESLintブランドの個性を反映することができるのです。
+        - >
+          ブランドの中心となるカラーはボタン、リンク、入力コンポーネントなど、すべてのインタラクティブな要素で使用されます。
+          これは私たちのロゴに由来しています。既存の2色を使って、独自の一次色調を作り出しています。
+      brand_palette: ブランド パレット
+      colors:
+        primary: プライマリ
+        white: 白色
+        neutral: ニュートラル
+  actions:
+    back_to_home: ホームページへ
+    browse_docs: ドキュメントを読む
+
+#------------------------------------------------------------------------------
+# Homepage
+#------------------------------------------------------------------------------
+
+homepage:
+  title: 解析対象のJavaScriptコードにおける問題発見と解消
+  description: >
+    ESLintはコードを静的に解析し、問題を迅速に発見します。
```

Since https://github.com/eslint/new.eslint.org/pull/193#discussion_r854381811 has been accepted, I think it is better to accept this suggestion as well.

```suggestion
    ESLintはコードを静的に解析し、迅速に問題を発見します。
```

Original: https://github.com/eslint/new.eslint.org/pull/193#discussion_r854355754